Intro Code

volts-inventory opened the pull request:
-Integrated michelle's old intro code
-Added entries in UI for DOB and Gender
-Adds UI information into user manager

volts-inventory committed:
-Integrated michelle's old intro code
-Added entries in UI for DOB and Gender
-Adds UI information into user manager

volts-inventory committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	app/build.gradle
#	app/src/main/AndroidManifest.xml

0queue commented:
Initial notes: style changes, remove unnecessary xml (layouts, random strings), restore color blind mode

0queue commented on app/src/main/AndroidManifest.xml:
```diff
@@ -1,27 +1,28 @@
         android:supportsRtl="true"
-        android:theme="@style/AppTheme.ColorBlind">
-        <activity android:name=".MainActivity">
+        android:theme="@style/AppTheme">
```
Should be AppTheme.ColorBlind

0queue commented on app/src/main/AndroidManifest.xml:
```diff
@@ -1,27 +1,28 @@
             android:enabled="true"
-            android:exported="false">
-        </service>
+            android:exported="false"></service>
```
Let's get rid of these empty tags while we're at it

0queue commented on app/src/main/java/edu/umd/cmsc436/mstestsuite/IntroActivity.java:
```diff
@@ -0,0 +1,190 @@
+        super.onCreate(savedInstanceState);
+        mUserManager = new UserManager(this.getApplicationContext());
+        if (mUserManager.getCurUserID() != null) {
+            Log.i("Login", "There was a user");
```
Get rid of unnecessary logs

0queue commented on app/src/main/java/edu/umd/cmsc436/mstestsuite/IntroActivity.java:
```diff
@@ -0,0 +1,190 @@
+        next.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                int current = getItem(+1);
```
That call to getItem(+1) can just be getItem(1)

0queue commented on app/src/main/java/edu/umd/cmsc436/mstestsuite/IntroActivity.java:
```diff
@@ -0,0 +1,190 @@
+        EditText dob = (EditText) findViewById(R.id.dob);
+        Spinner handSpin = (Spinner) findViewById(R.id.handedness_spinner);
+        Spinner genderSpin = (Spinner) findViewById(R.id.gender_spinner);
+        Log.i("Log", "USER Added");
```
(Log again)

0queue commented on app/src/main/res/layout/activity_intro.xml:
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Whole file could use auto-format (Ctrl-Alt-L, or something similar on Mac)

0queue commented on app/src/main/res/layout/activity_intro_1.xml:
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
(Auto format)

0queue commented on app/src/main/res/layout/activity_intro_2.xml:
```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```
(Auto format)

0queue commented on app/src/main/res/layout/activity_intro_3.xml:
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```
This file should be combined with activity_intro_1, so there are only two intro layouts

0queue commented on app/src/main/res/values/colors.xml:
```diff
@@ -2,7 +2,29 @@
+    <color name="colorAccent">#FFD90F</color>
     <color name="grayBackground">#9e9e9e</color>
 
+    <color name="screen1">#E3F2FD</color>
```
All of these new colors should probably be changed, to a black/white or very light gray, with the dots being colorAccent436 and colorAccentLight436

0queue commented on app/src/main/res/values/dimens.xml:
```diff
@@ -3,4 +3,10 @@
+
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
```
Only fab_margin is being used, and not even for a fab, so these should be removed

volts-inventory replied:
We need fab_margin

0queue commented on app/src/main/res/values/strings.xml:
```diff
@@ -10,4 +10,38 @@
+    <string name="handedness">Handedness:</string>
+    <string name="test_recyclerview_header">Practice Mode</string>
+
+    <string name="tap_start_text">Tap to start</string>
```
Unnecessary strings

volts-inventory replied:
we need handedness

0queue commented on app/src/main/AndroidManifest.xml:
```diff
@@ -1,27 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="edu.umd.cmsc436.mstestsuite">
+    package="edu.umd.cmsc436.mstestsuite">
```
Should also be auto formatted

0queue reviewed (changes requested):
Clean up of some leftovers from the other project, and then style changes.  Also, combine the last and first intro layouts, they don't need to be separate

volts-inventory committed:
Merge remote-tracking branch 'upstream/master'

volts-inventory committed:
-Fixed issues thomas had
-When switching to new user, will launch intro activity
-Removed redundant stuffs

0queue commented on app/src/main/java/edu/umd/cmsc436/mstestsuite/IntroActivity.java:
```diff
@@ -13,12 +13,16 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
```
Unused imports (Ctrl-Alt-O on the file to remove)

0queue commented on app/src/main/java/edu/umd/cmsc436/mstestsuite/IntroActivity.java:
```diff
@@ -35,9 +39,11 @@
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
```
Just one last style change: Change the status bar color to white with the line `getWindow().setStatusBarColor(ContextCompat.getColor(this, android.R.color.white));` (It looks really slick)

0queue commented on app/src/main/res/layout/activity_intro.xml:
```diff
@@ -1,40 +1,53 @@
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
```
Not sure why there's a Toolbar here, it can be deleted

0queue reviewed (changes requested):
Looks a lot better, thanks!  Just three tweaks and that'll be it!

volts-inventory committed:
Merge remote-tracking branch 'upstream/master'

volts-inventory committed:
-Removed unused imports
-Removed unused toobar
-Added white status bar
